apps/settings/locales/settings.en-US.properties

```diff
@@ -843,6 +843,9 @@ fxsync-collections-passwords=Passwords
 fxsync-legal=Legal
 fxsync-tos=Terms of Service
 fxsync-privacy=Privacy Notice
+fxsync-error-invalid-account=This account is inactive
+fxsync-error-invalid-account-explanation=You need to login in Firefox or Firefox for Android and retry
```

Comment:
nit: You need to first login in Firefox or Firefox for Android and then try again.